website/src/content/blog/roadmap-2024.md

```diff
@@ -0,0 +1,353 @@
+socialImage: ../../assets/social-logo.png
+---
+## Roadmap 2024
+
+We are thrilled to share what the Core Contributors and Maintainers would like to focus on in 2024.
+
+We want to remind the community that Biome is a project driven by the community, so we can't promise that all the ideas outlined below are going to be shipped.
+
+However, if you're excited about some aspects of the project, and you want to see some of them developed faster than others, you can help us in many ways:
+- [**Be involved in the project and the community**](https://github.com/biomejs). Help us to build those features.
+- [**Sponsor us**](https://opencollective.com/biome). Ask your company to sponsor us. Biome is so fast that can cut down CI times in your company, and save money. Performance is part of our mission. Plus, sponsorship is a good medium of _advertisement_ for your company.
+- [**Improve our documentation with ideas, recipes, or guides**](/guides/getting-started). Translate our documentation, help us to make Biome available to people who aren't proficient in English.
+
+
+### Preface
+
+The project is young and still can't compete against other giants such as Prettier, eslint, webpack, vite, esbuild, etc., although the recent events (sponsors, bounty challenge, Biome being a fork of Rome) showed that the users **have** interest in the project, and we showed those users that we have the tools for fulfilling a need.
+
+Moving small projects from ESLint/Prettier is easy, but moving **big** code bases is challenging and time-consuming. This is a big friction to Biome.
+
+Users have different needs though, so it will be impossible to satisfy all of them, so we want to make sure that they contribute to our project and embrace our philosophy.
+
+### Main area of focus
+
+1. Help users to move to Biome
+2. Increase Biome's languages, so Biome tools can span
+3. Deepen Biome's existing capabilities to offer more functionalities
+4. Plugins
+5. Transformations
+6. Community and content
+
+### Help users to move to Biome
+
+- Offer guides on our website to users who want to migrate from Prettier (CLI commands and configuration)
+- Offer guides on our website to users who want to migrate from ESlint (CLI commands and configuration)
+- Offer a section on our website that shows a mapping of the ESLint rules to our rules
+- Offer commands to ease the transition
+  - A command called `biome migrate prettier` that will read `.prettierrc` and `.prettierignore`, will update the `biome.json` file (or create it) with the configuration coming from the Prettier files.
+  - A command called `biome migrate eslint` that will read the JSON configuration of Eslint and the ignore file. There will be expectations and limitations.
+
+
+### Increase Biome's languages, so Biome tools can span
+
+CSS is going to be our next language of focus. HTML and Markdown will follow. Follow our [up-to-date page](/internals/language-support) to know the progress of our works.
```

Comment:
```suggestion
CSS is our next language of focus and is making good progress. HTML and Markdown will follow. Follow our [up-to-date page](/internals/language-support) to keep up with the progress of our work.
```
Slightly changed the phrasing to call out that CSS has already been started since it's already pretty far along for parsing and formatting.